### ISSUE TITLE
Update gitianbuild docs for bionic

### ISSUE THREAD

**Issue by marcus290** [navcoin/navcoin-core#547]
Since ec6d86d, Gitian uses Ubuntu Bionic as the base virtual image instead of Trusty.

Need to update our docs for bionic instead of trusty.

Also might need to update the dependencies to create the bionic image. For example, I needed to follow [these steps](https://gist.github.com/laanwj/c62e101bfd68718f0686926dfd10666b) to add further dependencies to my debian host.

**Comment by mxaddict** [comment]
@marcus290 yes, we need to update the docs, but we are no longer using LXC we instead use docker

It's much easier to setup and no longer needs a VM as long as you can run ruby + docker on host machine.

**Comment by mxaddict** [comment]
Basically you do this:
```
sudo apt install docker ruby
git clone https://github.com/devrandom/gitian-builder.git
git clone https://github.com/navcoin/navcoin-core.git
cd gitian-builder
bin/make-base-vm --docker --arch amd64 --suite bionic
USE_DOCKER=1 ./bin/gbuild -j<threads> --commit navcoin-core=<branch> --url navcoin-core=<repo_url> ../navcoin-core/contrib/gitian-descriptors/gitian-<os>.yml
```

**Comment by mxaddict** [comment]
Using LXC required a VM to run LXC vm inside.

The new process using docker can be run from host machine, so much easier to setup and less overhead requirement on performance.

**Comment by ChuanWeis** [comment]
I opened PR #970 to align the Gitian setup docs and helper script with the current descriptor suites: https://github.com/navcoin/navcoin-core/pull/970

**Comment by aguycalled** [comment]
this repo is deprecated, please have a look at nav-io/navio-core for the latest state of the project
